src/scheduler/data_types.h

```diff
@@ -1150,6 +1150,7 @@ struct config
 	unsigned resv_conf_ignore:1;  /* if we want to ignore dedicated time when confirming reservations.  Move to enum if ever expanded */
 	unsigned allow_aoe_calendar:1;        /* allow jobs requesting aoe in calendar*/
 	unsigned logstderr:1;               /* log to stderr as well as log file */
+	unsigned opt_bf_fuzzy_time_set:1;	/* sched config opt_backfill_fuzzy_time is set */
```

Comment:
That's what I had done before, and it created a catch-22 situation. We need to know inside init_sc_attrs whether sched_config's opt_backfill_fuzzy_time has been set or not, otherwise we won't be able to set it to the default (BF_DEFAULT) correctly. Let me know what you think.

I'll remove logstderr thanks

Reply:
I'm a little confused.  If opt_backfill_fuzzy_time is set to BF_DEFAULT, then why can't it be used as the default from that point forward?

Reply:
Consider the case when pbs_sched comes up, we first call schedinit(), which, if opt_backfill_fuzzy_time has been set in sched_config, will set sc_attrs.opt_backfill_fuzzy to that time. Then, when sched receives its first command from server, it calls init_sc_attrs(), if we set sc_attrs.opt_backfill_fuzzy to BF_DEFAULT there then we'll overwrite the sched_config value.

Reply:
I don't think I was clear.  You can't get rid of conf.opt_backfill_fuzzy_time.  Set this variable to BF_DEFAULT in init_config().  If opt_backfill_fuzzy_time is in the config file, set it to that.  In init_scattrs(), set scattrs.opt_backfill_fuzzy_time to conf.opt_backfill_fuzzy_time.  If opt_backfill_fuzzy is not set by in the sched object, we'll use that.

Reply:
Ah, ok, I see your point. I was going to do that earlier, but I thought that it would be more clear if we set sc_attrs.opt_backfill_fuzzy directly in init_config() instead of having 2 different globals for it, which are set by different code paths. So, I've removed conf.opt_backfill_fuzzy_time and now only have a flag which says whether sc_attrs.opt_backfill_fuzzy was set by init_config() or not. I thought that'd be more clear, let me know what you think.

Reply:
parse_config* not init_config, sorry for the confusion